test/issues/issues-test.js

```diff
@@ -508,4 +508,14 @@ describe("issues", function () {
             sinon.assert.calledWith(spy, secondObj);
         });
     });
+
+    describe("#1796 - cannot stub Array.prototype.sort", function () {
+        it("it should not fail with RangeError", function () {
+            refute.exception(function () {
+                var stub = sinon.stub(Array.prototype, "sort");
+                [1, 2, 3].sort();
```

Comment:
This should be the only line that we are concerned with not throwing an exception.

Reply:
Correct, I've pushed a commit that fixes that